libbeat/kibana/client.go

```diff
@@ -283,9 +293,31 @@ func (client *Client) GetDashboard(id string) (common.MapStr, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error removing index pattern: %+v", err)
 	}
+
+	// remove unsupported chars
+	objects := result["objects"].([]interface{})
+	for _, obj := range objects {
+		o := obj.(common.MapStr)
+		for _, key := range responseToDecode {
+			decodeValue(o, key)
+		}
+	}
+	result["objects"] = objects
 	return result, nil
 }
 
+// error is not propagated, because it can fail if the key is missing
```

Comment:
For the record, I prefer @ruflin solution. I am not changing it here, because my PR will need a rebase after his is merged.

Reply:
Rebased, resolved.